spec/swagger_v2/api_swagger_v2_detail_spec.rb

```diff
@@ -104,7 +106,7 @@ def app
     specify do
       expect(subject['paths']['/use_gfm_detail']['get']).to include('description')
       expect(subject['paths']['/use_gfm_detail']['get']['description']).to eql(
-        "<h1 id=\"this-returns-something\">This returns something</h1>\n<p># Burgers in Heaven</p>\n\n<blockquote>\n  <p>A burger doesn’t come for free</p>\n</blockquote>\n\n<p>If you want to reserve a burger in heaven, you have to do<br />\nsome crazy stuff on earth.</p>\n\n<pre><code>def do_good\nputs 'help people'\nend\n</code></pre>\n\n<ul>\n  <li><em>Will go to Heaven:</em> Probably</li>\n  <li><em>Will go to Hell:</em> Probably not</li>\n</ul>"
+        "> A burger doesn't come for free\n\nIf you want to reserve a burger in heaven, you have to do\nsome crazy stuff on earth.\n\n```\ndef do_good\nputs 'help people'\nend\n```\n\n* _Will go to Heaven:_ Probably\n* _Will go to Hell:_ Probably not\n"
```

Comment:
please revert it, cause a MArkdown formmater is set (https://github.com/Mapotempo/grape-swagger/blob/a41387c14f667bd88fd63addf04f949e1d8e7673/spec/swagger_v2/api_swagger_v2_detail_spec.rb#L92), means it should be transformed to HTML